### PR TITLE
Homework2: Dzmitry Bachkouski

### PR DESCRIPTION
**Pull request by Dzmitry89**
Please check my Homework2.

### CODE REVIEW

File: src/HW_2_Additional_tasks.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    while i <= var**0.5:
+        if var % i == 0:
+            print(var, "-составное число")
+            break
+        i += 1
+    else:
+        print(var, "-простое число")
+for i in range(var):
+    p = i + 1
+    print(p, '**3=', p ** 3)
```
Comment (belovmd) [inline src/HW_2_Additional_tasks.py:23]: what is it? please add task description

File: src/HW_2_Additional_tasks.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+# task_2 Palindrom
```
Comment (belovmd) [inline src/HW_2_Additional_tasks.py]: please separate tasks with blanc lines

File: src/HW_2_Additional_tasks.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+x = int(input("Введите натуральное число x="))
+y = x
+z = 0
```
Comment (belovmd) [inline src/HW_2_Additional_tasks.py]: x y z unclear naming

File: src/HW_2_Additional_tasks.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+while y > 0:
+    z = z * 10 + y % 10
+    y = y // 10
+if z == x:
+    print(x, "является палиндромом")
+else:
+    print(x, "не является палиндромом")
+# task_3 Fibonacci numbers
```
Comment (belovmd) [inline src/HW_2_Additional_tasks.py]: please separate tasks with lines

File: src/HW_2_Additional_tasks.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    print(fib1)
+elif n == 2:
+    print(fib1)
+    print(fib2)
+else:
+    print(fib1)
+    print(fib2)
+    for i in range(n - 2):
+        fib12 = fib1 + fib2
+        fib1 = fib2
```
Comment (belovmd) [inline src/HW_2_Additional_tasks.py]: a, b = b, a+b

File: src/HW_2_task_2_from_the_site.py [hunk 6]
```diff
@@ -0,0 +1,71 @@
+﻿# Say Hi
+def say_hi(name, age):
+    return "Hi. My name is {name} and I'm {age} " \
+           "years old".format(name=name, age=age)
+
+
+print(say_hi("alex", 32))
+
+
+# DJ Polycarpus
```
Comment (belovmd) [inline src/HW_2_task_2_from_the_site.py]: please put tasks descriptions

File: src/HW_2_task_2_from_the_site.py [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+        if k != 0:
+            s *= k
+    return s
+
+
+print(checkio(123405))
+
+"""
+Дан кусок текста. Соберите все заглавные буквы в одно
+слово в том порядке как они встречаются в куске текста.
```
Comment (belovmd) [inline src/HW_2_task_2_from_the_site.py:92]: where is it?

File: src/HW_2_task_3_long_word.py [hunk 8]
```diff
@@ -0,0 +1,12 @@
+my_string = "I am not a magician, I am just learning!!!"
+my_string = my_string.split()
+i = 0
+for word in my_string:
+    mod_word = word.strip(".,?!;:")
```
Comment (belovmd) [inline src/HW_2_task_3_long_word.py]: we have other punctuation marks

File: src/HW_2_task_3_long_word.py [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+    dl = len(mod_word)
```
Comment (belovmd) [inline src/HW_2_task_3_long_word.py]: i, dl - unclear naming

**Review by belovmd — COMMENTED** [review]
Please take a look at these issues

File: src/HW_2_Additional_tasks.py (hunk 1 above)
Comment (Dzmitry89) [reply src/HW_2_Additional_tasks.py:23]: Sorry, task description added

File: src/HW_2_Additional_tasks.py (hunk 2 above)
Comment (Dzmitry89) [reply src/HW_2_Additional_tasks.py]: blanc lines added

File: src/HW_2_Additional_tasks.py (hunk 3 above)
Comment (Dzmitry89) [reply src/HW_2_Additional_tasks.py]: names changed

File: src/HW_2_Additional_tasks.py (hunk 4 above)
Comment (Dzmitry89) [reply src/HW_2_Additional_tasks.py]: blanc lines added

File: src/HW_2_Additional_tasks.py (hunk 5 above)
Comment (Dzmitry89) [reply src/HW_2_Additional_tasks.py]: changed

File: src/HW_2_task_2_from_the_site.py (hunk 6 above)
Comment (Dzmitry89) [reply src/HW_2_task_2_from_the_site.py]: task description added

File: src/HW_2_task_2_from_the_site.py (hunk 7 above)
Comment (Dzmitry89) [reply src/HW_2_task_2_from_the_site.py:92]: Sorry, code added

File: src/HW_2_task_3_long_word.py (hunk 8 above)
Comment (Dzmitry89) [reply src/HW_2_task_3_long_word.py]: import function from module "string"

File: src/HW_2_task_3_long_word.py (hunk 9 above)
Comment (Dzmitry89) [reply src/HW_2_task_3_long_word.py]: names changed

File: src/HW_2_task_3_long_word.py [hunk 10]
```diff
@@ -0,0 +1,13 @@
+import string
```
Comment (belovmd) [inline src/HW_2_task_3_long_word.py:1]: missed task description

File: src/HW_2_task_2_from_the_site.py [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+uppercase English letters, the string's length doesn't exceed 200 characters
+Output: Return the words of the initial song that Polycarpus used to make
+a dubsteb remix. Separate the words with a space.
+"""
+
+
+def song_decoder(song: str) -> str:
+    song = song.replace("WUB", " ")  # заменяем в предложении WUB на " "
+    char_index = 0
+    while char_index < len(song) - 1:  # цикл удаляет все повторяющиеся пробелы
```
Comment (belovmd) [inline src/HW_2_task_2_from_the_site.py:36]: strip?

**Review by belovmd — COMMENTED** [review]
8,5

**Pull request closed without merging**